packages/mdc-linear-progress/foundation.ts

```diff
@@ -98,7 +101,7 @@ export default class MDCLinearProgressFoundation extends MDCFoundation {
     this.adapter_.addClass(cssClasses.CLOSED_CLASS);
   }
 
-  setScale_(el, scaleValue) {
+  private setScale_(el: HTMLElement, scaleValue: number) {
     const value = 'scaleX(' + scaleValue + ')';
```

Comment:
nit/optional: I like to use the backticks for this, I find it more readable:

```ts
`scaleX(${scaleValue})`
```

Reply:
Done, thanks!